techlibs/quicklogic/synth_quicklogic.cc

```diff
@@ -244,7 +248,9 @@ struct SynthQuickLogicPass : public ScriptPass {
                 techMapArgs += "_ffs_map.v";
             }
 
-            run("techmap " + techMapArgs);
+            if(!openfpga) {
+                run("techmap " + techMapArgs);
+            }
```

Comment:
don't we need any techmap for openfpga?

Reply:
In openfpga Caraval architecture, by default latches are inferred  and my objective is to generate blif that can directly be used in openfpga flow. So, this doesn't require mapping FF and the above marked code is for tech-mapping ff.

Reply:
ok